src/GSvar/MainWindow.h

```diff
@@ -544,6 +550,7 @@ public slots:
 
     //current server version (if in client-server mode)
     QString server_version_;
+    QThreadPool igv_init_thread_pool_;
```

Comment:
Remove and use BackgroundWorkerBase to run backgrund jobs, similar to NGSDCacheInitializer